### ISSUE TITLE
How to get results reported in paper for cifar-10?

### ISSUE THREAD

**Issue by arunpatala** [hongyi-zhang/mixup#1]
Thanks for a very interesting paper!! I was trying out your code and was not able to get the results posted in paper but got the results shown on GitHub page.

CIFAR10 PreAct ResNet-18 results of 3.9% are reported in the paper. But the GitHub page gives it 4.24%. Are there any other setting to change like epochs, weight decay etc? 

Thanks

**Comment by hongyi-zhang** [comment]
Hi, thanks for your interest! For unknown reasons we were not able to reproduce the 3.9% result (at least with the current [pytorch-cifar](https://github.com/kuangliu/pytorch-cifar) repo). We already updated the ICLR submission with the new numbers; will do so for the arXiv version as well.

Best,
Hongyi

**Comment by arunpatala** [comment]
Thanks. I was able to get 3.9% by running for longer epochs. I tried with 400 epochs.

**Comment by hongyi-zhang** [comment]
Thanks for your input! Closing the issue.
